src/ECS/Components/Creature.h

```diff
@@ -0,0 +1,27 @@
+/*****************************************************************************
+ * Copyright (c) 2018-2022 openblack developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/openblack/openblack
+ *
+ * openblack is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#include <string>
+
+#include "3D/CreatureBody.h"
+#include "Creature.h"
+#include "Enums.h"
+
+namespace openblack::ecs::components
+{
+
+struct Creature
+{
+	std::string owner;
+	CreatureType species;
+	std::string mindName;
```

Comment:
Is it possible to make an enum for this? Does this value map to anything in info.dat?

Reply:
I really don't know enough about info.dat. Not inspected the file for creature mind stuff

Reply:
What sort of values come out?

Reply:
I see
* "ComputerControlledCreature"
* "COMPUTERCONTROLLEDCREATURE"

Reply:
This seems to be for `Scripts\CreatureMind\ComputerControlledCreature`
The directory has other minds and I guess mods can add others
* ComputerControlledCreature
* KhazarCreature
* LethysCreature
* NemesisCreature

If I grep for `KhazarCreature`, I find references in my save files and in Scripts/Quests/challenge.chl

Reply:
I've started decyphering erc files here:
https://github.com/bwrsandman/openblack/tree/erc